assets/js/blocks/breadcrumbs/block.json

```diff
@@ -0,0 +1,44 @@
+{
+	"name": "woocommerce/breadcrumbs",
+	"version": "1.0.0",
+	"title": "Store Breadcrumbs",
+	"description": "Enable customers to keep track of their locations within the store and navigate back to parent pages.",
+	"category": "woocommerce",
+	"keywords": [ "WooCommerce" ],
+	"textdomain": "woo-gutenberg-products-block",
+	"usesContext": [ "postId", "postType" ],
+	"attributes": {
+		"contentJustification": {
+			"type": "string"
+		},
+		"fontSize": {
+			"type": "string",
+			"default": "small"
+		}
+	},
+	"supports": {
```

Comment:
Based on the planned native breadcrumbs supports: https://github.com/WordPress/gutenberg/pull/32500/files